src/data_sources.py

```diff
@@ -214,7 +214,7 @@ class MetadataRewritePreprocessor(preprocessor.DaskMultiFilePreprocessor):
     :class:`ExplicitFileDataSourceConfigEntry` objects in the \config_by_id
     attribute of :class:`ExplicitFileDataSource`.
     """
-    _file_preproc_functions = []
+    file_preproc_functions = []
```

Comment:
Could you submit modifications to the variable definitions in a separate PR? The file_preproc_functions changes don't appear to cause problems like the _functions change did, but it would be better to limit this PR to docs changes.

Reply:
Resolved in [130e994](https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/372/commits/130e994fcaf3a78a0ec4188e2a375e4c0f5cc5d9), see https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/372#discussion_r897516962